go.mod

```diff
@@ -73,46 +73,46 @@ require (
 	github.com/muesli/reflow v0.3.0
 	github.com/muesli/termenv v0.16.0
 	// pin v0.0.5
-	github.com/olekukonko/tablewriter v0.0.5
+	github.com/olekukonko/tablewriter v1.0.9
```

Comment:
Do we really want to use this. The GitHub repo says "Warning: Version v1.0.0 contains missing functionality and should not be used."

Reply:
I reverted this to 0.0.5 because we also have this dependency in the server.